chatterbot/storage/django_storage.py

```diff
@@ -1,7 +1,5 @@
-import json
 from chatterbot.storage import StorageAdapter
 
-
```

Comment:
And also the response and statements are still reversed in django_storage adapter